pkg/controller/batchrelease/batchrelease_plan_executor.go

```diff
@@ -89,30 +95,23 @@ func (r *Executor) executeBatchReleasePlan(release *v1alpha1.BatchRelease, newSt
 
 	case v1alpha1.RolloutPhaseHealthy:
 		// verify whether the workload is ready to execute the release plan in this state.
-		var verifiedDone bool
-		verifiedDone, err = workloadController.VerifyWorkload()
-		switch {
-		case err != nil:
-			setCondition(newStatus, v1alpha1.VerifyingBatchReleaseCondition, v1.ConditionFalse, v1alpha1.FailedBatchReleaseConditionReason, err.Error())
-		case verifiedDone:
-			newStatus.Phase = v1alpha1.RolloutPhasePreparing
-			result = reconcile.Result{RequeueAfter: DefaultDuration}
-			setCondition(newStatus, v1alpha1.PreparingBatchReleaseCondition, v1.ConditionTrue, "", "BatchRelease is preparing for progress")
-		}
+		newStatus.Phase = v1alpha1.RolloutPhasePreparing
+		utils.SetCondition(newStatus, v1alpha1.PreparingBatchReleaseCondition, v1.ConditionTrue, "", "BatchRelease is preparing for progress")
+		fallthrough
 
 	case v1alpha1.RolloutPhasePreparing:
```

Comment:
Is it possible to simplify the state, init, healthy, preparing is actually the init state?